.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,36 +4,55 @@ about: Create a report to help us improve
 
 ---
 
-<!--
-PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+<!-- Before submitting an issue, please consult our troubleshooting guide (http://ionicframework.com/docs/troubleshooting/) and developer resources (http://ionicframework.com/docs/developer-resources/) -->

```

Comment:
We probably shouldn't link back to v3 docs since this is now only the v4 repository